lib/core/network/api/endpoints.dart

```diff
@@ -1,21 +1,49 @@
+import 'dart:convert';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+
+enum Networks { telos, telosTestnet, eos, eosTestnet }
+
 class Endpoints {
   Endpoints._();
 
-  // base url - the load balancer takes care of available features on a given node, such as history, etc
-  static const String baseUrl = 'http://mainnet.telos.net';
+  static Map<String, dynamic> getNetworkConfig({String? network}) {
+    final remoteConfig = FirebaseRemoteConfig.instance;
+    network = network ?? Networks.telos.name;
+    final conf = json.decode(remoteConfig.getValue('networks').asString());
+    if (conf[network] == null) {
+      throw 'Unknown network: $network';
+    }
+    return conf[network];
+  }
 
-  // Node to push transactions - should be a fast server
-  static const String pushTransactionNodeUrl = 'http://telos.greymass.com';
+  // base url - read URL
+  // network: default is Telos mainnet
+  static String baseUrl({String? network}) {
+    final networkConfig = getNetworkConfig(network: network);
+    final endpoint = networkConfig['endpoint'];
```

Comment:
this would be encapsulated inside that service 
RemoteConfigService 

Then this service can be injected in the module

Reply:
Ok - added RemoteConfigService